.circleci/define_build_environment_variables

```diff
@@ -1,3 +1,4 @@
 #!/bin/sh -e
 safe_git_branch=${CIRCLE_BRANCH//\//-}
 short_sha="$(git rev-parse --short=7 $CIRCLE_SHA1)"
+ECR_DEPLOY_IMAGE="$ECR_DOCKER_REPO_BASE:$safe_git_branch.$short_sha"
```

Comment:
_not a blocker_
🙋‍♂ `ECR_DOCKER_REPO_BASE` and `ECR_ENDPOINT` seem to be pointing to the same value.

Reply:
Looking back on this, I realise they are not the same, as `REPO_BASE` has the application image. However, that might be good to spell out explicitly.